src/main/resources/us/kbase/templates/module_readme_test.vm.properties

```diff
@@ -1,8 +1,2 @@
 This directory should contain scripts and files needed to test your module's code.
- 
-#if($example)
-Sample client tests in Python, Perl, and Java have been provided and may be run
-using their respective client test files. Optionally, all tests can be run with
-the `run_all_client_tests.sh` script. Note that these require your module's 
-server code to be running.
-#end
```

Comment:
I have no idea what this is talking about and have never seen client tests. Looking at `module_test_all_clients.vm.properties`, it clearly hasn't worked since auth2 and I'm not sure if it worked before then